app/routes/index.js

```diff
@@ -1,5 +1,52 @@
-module.exports = function(app) {
+var validUrl = require('valid-url');
+var normalizeUrl = require('normalize-url');
+var originalURLHandler = require('../api/originalURLHandler.js');
+var shortURLHandler = require('../api/shortURLHandler.js');
+
+var shortURLRegExp = new RegExp('^[0-9]{7}$', 'g');
+
+module.exports = function(app, dbURL) {
 	app.get('/', function(req,res) {
 		res.render('index');
 	});
+
+	app.get('/*', function(req,res) {
```

Comment:
Ah, that is a much better idea than what I was thinking of doing next. I was planning on putting the regex into the app.get's scope and then cleaning up the routes in general. I'm going to dig into this when I'm working with promises.